care/facility/models/bed.py

```diff
@@ -65,3 +65,10 @@ class ConsultationBed(BaseModel):
     start_date = models.DateTimeField(null=False, blank=False)
     end_date = models.DateTimeField(null=True, blank=True, default=None)
     meta = JSONField(default=dict, blank=True)
+
+    class Meta:
+        constraints = [
```

Comment:
Let's take a case where a patient starts off in a regular bed, moves to an ICU, and then moves back to the normal bed, this is practically possible right?

Reply:
The correct solution should check if there is any patient currently using the bed.

Reply:
So, do you suggest I modify the api route when assigning patients to bed *along with* the added constraints?